fence/blueprints/google.py

```
@@ -180,19 +182,27 @@ def patch(self, id_):
             )
             return msg, 403
 
-        payload = flask.request.get_json() or {}
-        project_access = payload.get('project_access')
+        try:
+            payload = flask.request.get_json()
```

Specifically with `get_json`, you can avoid the `except` by passing `silent=True`, which silences errors:
```python
payload = flask.request.get_json(silent=True) or {}
```

---

ooh, fancy. changing to that